proto/sysinfo.proto

```diff
@@ -479,3 +479,23 @@ message HardwareInfo {
     }];
 
 }
+
+// CronTab Entry
+message CronTabEntry{
```

Comment:
I thought about that, but it is possible that they aren't always ints due to the allowed special chars